### DISCUSSION TITLE
I can't format values to BRL currency

### DISCUSSION THREAD

**Discussion by Karytonn** [beholdr/maska#190] [Q&A]
### Describe the bug

I can't format values above 4 decimal places using the BRL currency (pt-BR)

Ex.: R$ 45.660,00 - is not possible

### Steps to reproduce

Below is a code example

### Reproduction link

Link to online sandbox with reproduction code: [Nuxt](https://stackblitz.com/edit/nuxt-starter-xpjzvb?file=public%2Fmaska.ts)

**Comment by Gustavopereira42** [comment]
Hi, I'm not sure if you still need this, but following your code I fixed it by changing the preProcess function to this:
```preProcess: (val) => val.replace(/[.]/g, '').replace(/[,]/g, '.').replace(/[$,]/g, '')```.

These are my complete options:
```
{
  preProcess: (val) => val.replace(/[.]/g, '').replace(/[,]/g, '.').replace(/[$,]/g, ''),
  postProcess: (val) => {
    if (!val) return '';

    const sub = 3 - (val.includes('.') ? val.length - val.indexOf('.') : 0)
    return Intl.NumberFormat('pt-BR', {
    style: 'currency',
      currency: 'BRL'
    }).format(val).slice(0, sub ? -sub : undefined);
  },
}
```

**Reply by jotape-exe** [reply]
Valeu @Gustavopereira42 ❤️

**Reply by hamonCordova** [reply]
In this code i need to type the comma for the decimal digits. What about when the user type, it just goes with the decimal?
Eg: 0,00 => 0,01 => 0,12 => 1,23 => 12,34 ...

**Comment by emaia** [comment]
I do something like this https://stackblitz.com/edit/nuxt-starter-iyzrab?file=app.vue

```vue
<template>
        <input
          type="text"
          id="renda"
          name="Renda"
          placeholder="Renda bruta"
          aria-placeholder="Renda bruta"
          v-maska:[options]
          data-maska-tokens="9:[0-9]:repeated"
          data-maska="9.99#,##"
        />
</template>

<script setup>
const options = {
  reversed: true,
  preProcess: (val) => val.replace('R$ ', ''),
  postProcess: (val) => val ? `R$ ${val}` : '',
};
</script>

```
